demos/scipp-introduction-for-imaging.ipynb

```diff
@@ -374,7 +374,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "x = sc.Variable(['x'], values=np.linspace(-np.pi, np.pi, 20))\n",
+    "x = sc.Variable(dims=['x'], values=np.linspace(-np.pi, np.pi, 20))\n",
```

Comment:
`sc.linspace`

Reply:
I would vote to leave this one because we are illustrating compatibility with `numpy`

Reply:
Agreed. I didn't note the section we had this in.